src/projects/republic/MetadataPanel.tsx

```diff
@@ -173,6 +176,18 @@ function ResolutionMetadata(props: { annotations: AnnoRepoAnnotation[] }) {
 
   return (
     <>
+      <ProvenanceButton
+        className="d-block absolute right-5"
```

Comment:
Toen ik gisteravond naar de branch keek, zag ik dat de Button altijd rechtsboven blijft hangen in het metadatapaneel wanneer je scrollt. Heeft mogelijk iets met de CSS classes te maken?

Reply:
Fixed in [efee1a2](https://github.com/knaw-huc/textannoviz/pull/275/commits/efee1a27ee8be15bc3c0cc9706660244265033d2)